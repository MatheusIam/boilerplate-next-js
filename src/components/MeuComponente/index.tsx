import * as S from './styles'

const MeuComponente = () => (
  <S.Wrapper>
    <h1>MeuComponente</h1>
  </S.Wrapper>
)

export default MeuComponente
