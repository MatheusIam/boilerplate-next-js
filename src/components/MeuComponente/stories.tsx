import { Meta, StoryObj } from '@storybook/react'
import MeuComponente from '.'

export default {
  title: 'MeuComponente',
  component: MeuComponente
} as Meta

export const Default: StoryObj = {}
