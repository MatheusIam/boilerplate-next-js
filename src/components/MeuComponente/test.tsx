import { render, screen } from '@testing-library/react'

import MeuComponente from '.'

describe('<MeuComponente />', () => {
  it('should render the heading', () => {
    const { container } = render(<MeuComponente />)

    expect(
      screen.getByRole('heading', { name: /MeuComponente/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
