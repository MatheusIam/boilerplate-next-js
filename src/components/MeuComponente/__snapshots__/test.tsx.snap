// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MeuComponente /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    MeuComponente
  </h1>
</main>
`;
